posthog/hogql/database/schema/cohort_people.py

```diff
@@ -27,7 +27,7 @@ def select_from_cohort_people_table(requested_fields: dict[str, list[str | int]]
     from posthog.models import Cohort
 
     cohort_tuples = list(
-        Cohort.objects.filter(is_static=False, team__project_id=project_id)
+        Cohort.objects.filter(is_static=False, team__project_id=project_id, deleted=False)
```

Comment:
Does every cohort have `False` by default or is it `null` when not deleted?

If the latter, then you should instead add `deleted=True` to the `exclude` call below instead